lib/dai-plugin-mcd/test/CdpManager.spec.js

```diff
@@ -50,6 +50,13 @@ test('getCdp looks up ilk', async () => {
   expect(sameCdp.ilk).toEqual(cdp.ilk);
 });
 
+test('getCombinedEventHistory', async () => {
+  const proxy = await maker.currentProxy();
+  const events = await cdpMgr.getCombinedEventHistory(proxy);
+  console.log('events', events);
+  //add an expect here
```

Comment:
While we wait for the settling of the shifting sands of having vulcanize available for testing locally & on kovan, one thing you could do here is mock out the part of the code that makes the actual network request, and substitute in dummy data instead. That way you can test that all of the code that handles the response behaves correctly with a known response. e.g.:
```js
const mockFn = jest.fn(async () => dummyData);
maker.service(QUERY_API).methodBeingCalledInThisTest = mockFn;
const events = await cdpMgr.getCombinedEventHistory(proxy);
expect(mockFn).toBeCalled();
expect(events).toEqual(formattedDummyData);
```

And this is also just good testing practice overall, so long as you have other tests that exercise the actual API. (The only risk of course is of forgetting to change the dummy API response if the actual API changes.)